### PR TITLE
Unchumped wonder closet.

### PR DESCRIPTION
**Pull request by Allspice**
16 sets. Dubious balance. SUGGESTIONS STILL WELCOME.

:cl:
* tweak: Add some worthwhile stuff to the wonder closet maybe perhaps

### CODE REVIEW

**Comment by otatoh** [comment]
More ideas: dracula coat, hos/warden/officer alt jackets, darkholme suit (also useful to falseflag traitor items).

**Comment by ShiftyRail** [comment]
If the wonder closet has a guaranteed chance of gettting the HoS longcoat (which is RNG-based on main station and has 50% of spawning, 50% of getting a duplicate of the normal trenchcoat), then traders are my best friends

File: code/game/objects/items/trader.dm [hunk 1]
```diff
@@ -58,8 +58,24 @@
...
-
+	var/wonder_whitelist = list(
+	/obj/item/clothing/mask/morphing/corgi,
+	/obj/item/clothing/under/rank/vice,
+	/obj/item/clothing/shoes/clown_shoes/advanced,
+	list(/obj/item/clothing/suit/space/clown, /obj/item/clothing/head/helmet/space/clown),
+	/obj/item/clothing/shoes/magboots/magnificent,
+	list(/obj/item/clothing/suit/space/plasmaman/bee, /obj/item/clothing/head/helmet/space/plasmaman/bee),
+	list(/obj/item/clothing/head/celtic, /obj/item/clothing/suit/armor/ice),
```
Comment (MadmanMartian) [inline code/game/objects/items/trader.dm]: Celtic crown's supposed to be high-tier unique vault loot.

**Comment by AI-Is-Rogue** [comment]
How about the lich crown and lich robe/tattered robe.

**Comment by Dacendeth** [comment]
How about all the winter coats?

**Comment by DamianX** [comment]
Please look into the suggestions posted here

**Comment by YertBogan** [comment]
Who among the player base even knows all of the items that are worth including? The whole fun of the wardrobe was that you didn't know what you were going to get. How do I suggest what I don't know exists?

**Comment by MadmanMartian** [comment]
every vox trader, as all 2 of them frequently hunt for vaults

**Comment by Allspice** [comment]
I removed the celt stuff and added the lichwear, sure. Darkholme's already in the theatre vend, could add Sundowner but that can already spawn for the HOS and might make the closet too armour-heavy, and vamp basically ``sucks dick".

**Pull request merged**